roles/ci_gen_kustomize_values/README.md

```diff
@@ -33,6 +33,8 @@ with a message.
 * `cifmw_ci_gen_kustomize_values_userdata_b64`: (List) Base64 encoded list of data to combine in the generated output.
   Defaults to `[]`.
 * `ci_gen_kustomize_fetch_ocp_state`: (Boolean) If true it enables generating CI templates based on the OCP state. Defaults to `true`.
+* `cifmw_ci_gen_kustomize_values_storage_class_prefix`: (String) Prefix for `storageClass` in generated values.yaml files. Defaults to `"lvms-"` only if `cifmw_lvms` is True, otherwise it defaults to `""`. The prefix is appended to the `cifmw_ci_gen_kustomize_values_storage_class`. It is not recommended to override this value, instead set `cifmw_lvms` True or False.
```

Comment:
do we have any use-case where we would want to change the prefix? If not we probably do not need to expose this as a parameter and just define it internally when `cifmw_lvms` is true

Reply:
We do need `cifmw_ci_gen_kustomize_values_storage_class`. Previously it was not there and `local-storage` was hard-coded in multiple spots.

I'm relying on its prefix variable to always being defined. Do you want to provide another way to solve the problem of getting the right storage class name?

Or do you think I should not document the variable in the readme though I'd keep it in the defaults file? I could prefix it with a `_` to imply that it's private while at the same time keeping it. We could also spend a little time using some other method of getting the name we want. It's mainly there in order to have one flag control different outcomes in multiple roles.

Reply:
I think it's fine just not documenting it in the README if we don't expect users to need to set it. I see why it's needed, and the current approach makes sense to me, I just wanted to clarify if we expected it to have a value different than the default, thanks for the explanation

Reply:
we do have the prefix pattern already...

- cifmw_ci_gen_kustomize_values_snippets_dir_prefix
- cifmw_ci_gen_kustomize_values_dest_fname_prefix

so I thought it better to be transparent about how it works, especially if someone uses `cifmw_ci_gen_kustomize_values_storage_class` and is surprised to see a prefix there.

Reply:
that's a good point about transparency, and the explanation is thorough, I'm not opposed to keep it as it is now, just wanted to clarify

Reply:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/ci_gen_kustomize_values/README.md?plain=1#L23-L28

Reply:
OK, I think if we're going to keep the prefix we should document it for completeness.